content/docs/lists-and-keys.md

```diff
@@ -239,16 +238,58 @@ function Blog(props) {
 const posts = [
   {id: 1, title: 'Hello World', content: 'Welcome to learning React!'},
   {id: 2, title: 'Installation', content: 'You can install React from npm.'}
+];id: 2, title: 'Installation', content: 'You can install React from npm.'}
 ];
 ReactDOM.render(
   <Blog posts={posts} />,
   document.getElementById('root')
 );
 ```
  
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/NRZYGN?editors=0010)
+[**Wypróbuj na CodePen**](https://codepen.io/gaearon/pen/NRZYGN?editors=0010)
+ 
+Klucze służą Reaktowi jako wskazówka, ale nie są one przekazywane do twoich komponentów. Jeżeli potrzebujesz tych wartości w swoim komponencie, przekaż je jawnie poprzez właściwość z inną nazwą:
```

Comment:
Wygląda na to, że cała ta sekcja została skopiowana i wklejona poniżej. Jest przetłumaczona dwa razy.

Reply:
Poprawione. :)